mycity/platforms/amazon/models/en_US.json

```diff
@@ -1449,11 +1510,28 @@
             {
                 "id": "Elicit.Slot.1478934588421.914733191093",
                 "variations": [
+                    {
+                        "type": "PlainText",
+                        "value": "I will need your address for that. What's your address?"
```

Comment:
Alexa guides emphasize keeping things short and conversational. I would have these just requesting `"What's your address?"` since that is what I would expect someone to say to me